index.html

```diff
@@ -1,15 +1,21 @@
 <!DOCTYPE html>
-<html lang="en">
+<html lang='en'>
 
 <head>
-    <link rel="stylesheet" href="style.css">
-    <title>Youtube Downloader</title>
+  <link rel='stylesheet' href='./style.css' />
+  <title>Youtube Downloader</title>
 </head>
 
 <body>
-    <h1 class="heading">My Own Youtube Downloader !</h1>
-    <input class="URL-input" placeholder="https://www.youtube.com/watch?v=MtN1YnoL46Q"><button class="convert-button" id="mp4">Convert mp4</button><button class="convert-button" id='mp3'>Convert mp3</button>
-    <script src='script.js'></script>
+  <h1 class='heading'>Paste YouTube URL</h1>
+  <input class='URL-input' placeholder='https://www.youtube.com/watch?v=MtN1YnoL46Q' />
+  <select class='opt' name='' id=''>
```

Comment:
Here you have an empty/unused id and name attributes. you can either remove them or fill them with anything (I recommend removing the id attribute but keeping the name attribute but it doesn't make much difference)